sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -759,6 +759,10 @@ case class Range(
   }
 
   override def computeStats(): Statistics = {
+    if (!conf.cboEnabled) {
+      return Statistics(sizeInBytes = LongType.defaultSize * numElements)
+    }
+
```

Comment:
We need this?

Reply:
I removed the condition as we have all the statistics available for the operator. I have added the condition earlier because, if cbo is not enabled, other operators are not returning other statistics. Do we also need to remove histogram check as well?

Reply:
I removed histogram config check as well. Because histogram information is readily available. Also refactored the code and Its